Client/Helpers/SpotlightHelper.swift

```diff
@@ -0,0 +1,129 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+import Shared
+import CoreSpotlight
+import WebKit
+
+
+private let log = Logger.browserLogger
+private let browsingActivityType: String = "org.mozilla.firefox.browsing"
```

Comment:
Nit: For the App Bundle ID we use 
`org.mozilla.ios.firefox` instead of `org.mozilla.firefox`. Might make sense to include the `ios` here too.